lib/mpl_toolkits/mplot3d/axis3d.py

```diff
@@ -389,15 +389,15 @@ def d_interval(self):
 
     @d_interval.setter
     def d_interval(self, minmax):
-        return self.set_data_interval(*minmax)
+        self.set_data_interval(*minmax)
 
     @property
     def v_interval(self):
         return self.get_view_interval()
 
-    @d_interval.setter
```

Comment:
This was actually a bug.

Reply:
Would love to have a test for this...

Reply:
We should actually just deprecate them, as they are just mplot3d-specific synonyms for {get,set}_{view,data}_interval...

Reply:
Moved to #16053.